README.md

```diff
@@ -34,8 +34,13 @@ tab when the state is picked on the web page.
 
 ### towns.txt
 
-This is a space-delimited list of the towns in NY that you would like to set an
-alert for. eg. `WOODSTOCK ALBANY OSWEGO`.
+This is a newline-delimited list of the towns in NY that you would like to set an
+alert for. eg.
+
+`WOODSTOCK
+ALBANY
+NEW ROCHELLE
+OSWEGO`.
```

Comment:
Could you please make this a fenced code block?

[That's three backtick marks on a line with the contents inside of it. ](https://www.markdownguide.org/extended-syntax/)